docs/ansible/roles/grafana/getting-started.rst

```diff
@@ -0,0 +1,47 @@
+.. Copyright (C) 2019 Leonardo Bechea <leonardo.bechea@innobyte.com>
+.. Copyright (C) 2019 Alin Alexandru <alin.alexandru@innobyte.com>
+.. Copyright (C) 2019 DebOps <https://debops.org/>
+.. SPDX-License-Identifier: GPL-3.0-or-later
+
+Getting started
+===============
+
+.. contents::
+   :local:
+
+
```

Comment:
A __Default Configuration__ section should give a brief overview of the state the application will be in if deployed with defaults, especially:

* choices that were made that might differ from its out-of-the-box defaults
* things the end user must pay attention to before deploying to production